d2l-button.scss

```diff
@@ -1,4 +1,5 @@
 @import 'bower_components/d2l-colors/d2l-colors.scss';
+@import 'bower_components/d2l-typography/d2l-typography.scss';
```

Comment:
This should move `d2l-typography` from `devDependencies` to `dependencies` in `bower.json`.